apps/company_info/views.py

```diff
@@ -1,3 +1,64 @@
-# from django.shortcuts import render
+from typing import List
 
-# Create your views here.
+from django.db.models.query import Prefetch
+
+from rest_framework import status
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+from apps.company_info.models import CompanyName, Language, Tag
+from apps.company_info.serializers import CompanyDetailSerializer, CompanyListSerializer
+
+
+class CompanyCreateView(APIView):
```

Comment:
```suggestion
from rest_framework.permissions import AllowAny

class CompanyCreateView(APIView):
    permission_classes = [AllowAny]

```

참고:  https://ssungkang.tistory.com/entry/Django-APIView%EC%97%90-permission-%EC%A7%80%EC%A0%95%ED%95%98%EA%B8%B0